docs/sources/trackers/google-tag-manager/settings-template/index.md

```diff
@@ -88,7 +88,7 @@ See [here](/docs/resources/recipes-tutorials/recipe-anonymous-tracking/index.md)
 
 #### ID Service
```

Comment:
@greg-el 
What are the actions we need to do to change the template ? _Didn't change since I don't know how it should be done_

Reply:
Yep, all that needs to be done is the two "ID Service"s there to be changed.

Reply:
Nice, when the PR is up, let's change this file accordingly.